operators/hpe-greenlake-file-csi-operator/Makefile

```diff
@@ -1,7 +1,7 @@
 # Set these
-VERSION ?= 0.0.0
+VERSION ?= 2.6.4
 SOURCE_VERSION ?= $(VERSION)
-REPO_NAME ?= quay.io/myrepo
+REPO_NAME ?= quay.io/hpestorage
```

Comment:
These are meant to be parameters to make or set as environment variables. Don't set real values in the Makefile.